allennlp/models/constituency_parser.py

```diff
@@ -142,22 +142,32 @@ def forward(self,  # type: ignore
         Returns
         -------
         An output dictionary consisting of:
-        logits : ``torch.FloatTensor``
-            A tensor of shape ``(batch_size, num_spans, span_label_vocab_size)``
-            representing unnormalised log probabilities of the label classes for each span.
         class_probabilities : ``torch.FloatTensor``
             A tensor of shape ``(batch_size, num_spans, span_label_vocab_size)``
             representing a distribution over the label classes per span.
+        spans : ``torch.LongTensor``
+            The original spans tensor.
+        tokens : ``torch.LongTensor``
+            The token ids from the ``TextField``. Has shape (batch_size, num_tokens).
+         sentence_lengths : ``torch.LongTensor``, required.
```

Comment:
I think this is why sphinx failed. The error messages are awful. There's an extra space here.